tests/Unit/IO/Importers/Test_VolumeDataReaderActions.cpp

```diff
@@ -286,16 +302,18 @@ void test_actions(const std::variant<double, importers::ObservationSelector>&
 
 // [[TimeOut, 10]]
 SPECTRE_TEST_CASE("Unit.IO.Importers.VolumeDataReaderActions", "[Unit][IO]") {
-  test_actions<false>(0., false);
-  test_actions<false>(importers::ObservationSelector::First, false);
-  test_actions<false>(importers::ObservationSelector::Last, false);
-  CHECK_THROWS_WITH(test_actions<false>(0., false, true),
+  test_actions<false>(0., false, false);
+  test_actions<false>(0., true, false);
+  test_actions<false>(importers::ObservationSelector::First, false, false);
+  test_actions<false>(importers::ObservationSelector::Last, false, false);
+  CHECK_THROWS_WITH(test_actions<false>(0., false, false, true),
                     Catch::Matchers::ContainsSubstring("not supported"));
 
   for (const bool subcell_is_active : {false, true}) {
-    test_actions<true>(0., subcell_is_active);
-    test_actions<true>(importers::ObservationSelector::First,
+    test_actions<true>(0., false, subcell_is_active);
+    test_actions<true>(importers::ObservationSelector::First, false,
+                       subcell_is_active);
+    test_actions<true>(importers::ObservationSelector::Last, false,
```

Comment:
If this feature isn't supported for subcell or FD grids, does it error in a clear way? Meaning do we print a message somewhere like "this isn't supported for FD grids" or will it error in an obscure place like when we compare the inertial coordinates?